srcs/runners_manager/vm_creation/gcloud/GcloudManager.py

```diff
@@ -107,6 +109,9 @@ def configure_instance(
             metadata=Metadata(
                 items=[Items(key="startup-script", value=startup_script)]
             ),
+            scheduling=Scheduling(
+                preemptible=True
+            ),
             advanced_machine_features=AdvancedMachineFeatures(
```

Comment:
This should be a configuration at the pool level, located at the `config` level.